.kitchen.yml

```diff
@@ -112,9 +97,27 @@ provisioner:
           renew-by-default: true
 
 suites:
+  - name: default
+    provisioner:
+      pillars:
+        top.sls:
+          base:
+            '*':
+              - letsencrypt
+        letsencrypt.sls:
+          letsencrypt:
+            config: |
+              server = https://acme-v01.api.letsencrypt.org/directory
+              authenticator = standalone
+              agree-tos = True
+              renew-by-default = True
+            domainsets:
+              www:
+                - letsencrypttest.acme.com
```

Comment:
I'd change this to something like `letsencrypt-formula.example.com`, again, just in case.